dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/UiPluginServiceTest.java

```diff
@@ -82,6 +86,12 @@ public void testQueryPluginDetailById() {
         Mockito.when(pluginDefineMapper.queryDetailById(1)).thenReturn(pluginDefine);
         result = uiPluginService.queryUiPluginDetailById(1);
         Assertions.assertEquals(Status.SUCCESS, result.get("status"));
+
+        PluginDefine data = (PluginDefine) result.get("data");
+        String pluginParams = data.getPluginParams();
+        ArrayNode arrayNode = JSONUtils.parseArray(pluginParams);
+        String placeholder = arrayNode.path(0).path("props").path("placeholder").asText();
+        Assertions.assertEquals(2, JSONUtils.toMap(placeholder).size());
```

Comment:
We need to verify both `en` and `zh` in test.

Reply:
> We need to verify both `en` 和 `zh` in test.

done